core/BUILD.bazel

```diff
@@ -4,33 +4,16 @@ java_library(
     exports = ["//api"],
 )
 
-java_library(
-    name = "inprocess",
```

Comment:
Leave a forwarding library in place. Keep `name` and `visibility` and have it export //inprocess. Ditto for util.

Reply:
done